src/app/containers/ArticleTimestamp/index.test.jsx

```diff
@@ -263,6 +263,18 @@ describe('ArticleTimestamp', () => {
     expect(secondChild(renderedWrapper)).toMatch(regexUpdatedDate);
   });
 
+  it('should render one timestamp when firstPublished and lastPublished is the same day, and lastPublished is outside of the relative window', () => {
+    const renderedWrapper = renderedTimestamps(
+      <WrappedArticleTimestamp
+        firstPublished={1400140005000}
+        lastPublished={1400153537000}
```

Comment:
Would it be useful to add a comment here for what the UTC time is for each of these values?

Reply:
Akh yes it could be - there are loads of other undeclared ms values in that test file though. I could add them across the board? I think it's a good addition

Reply:
👍